hail_scripts/utils/clinvar.py

```diff
@@ -34,18 +34,19 @@ def download_and_import_latest_clinvar_vcf(genome_version: str) -> hl.MatrixTabl
         raise ValueError("Invalid genome_version: " + str(genome_version))
 
     clinvar_url = CLINVAR_FTP_PATH.format(genome_version=genome_version)
-    local_tmp_file_path = "/tmp/clinvar.vcf.gz"
-
-    subprocess.run(["wget", clinvar_url, "-O", local_tmp_file_path], check=True)
-    subprocess.run(["hdfs", "dfs", "-copyFromLocal", "-f", f"file://{local_tmp_file_path}", local_tmp_file_path])
-    clinvar_release_date = _parse_clinvar_release_date(local_tmp_file_path)
-    mt_contig_recoding = {'MT': 'chrM'} if genome_version == '38' else None
-    mt = import_vcf(local_tmp_file_path, genome_version, drop_samples=True, min_partitions=2000, skip_invalid_loci=True,
-                    more_contig_recoding=mt_contig_recoding)
-    mt = mt.annotate_globals(version=clinvar_release_date)
-
-    return mt
-
+    with tempfile.NamedTemporaryFile(delete=False, suffix='.vcf.gz') as local_tmp_file_path:
```

Comment:
would be better to name the variable `local_tmp_file` as it is the file object and not the path